patches/0001-Expose-Encoder.Canonical-and-MarshalCanonical-Handle.patch

```diff
@@ -380,21 +377,15 @@ index fca2a09..aaa79c2 100644
 +	} else {
 +		x.byIndex = f
 +	}
++	newM[t] = x
+ 	fieldCache.value.Store(newM)
```

Comment:
Am I missing something? `... q, r, s, t, u ...`. Doesn't `private` come before `public` so `{"keytype":"ecdsa-x509","keyval":{"public":"...","private":null}}`, is _not_ correctly lexicographically ordered.